projects/igniteui-angular/src/lib/grids/columns/interfaces.ts

```diff
@@ -51,6 +51,11 @@ export interface IFieldPipeArgs {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface IColumnPipeArgs extends IFieldPipeArgs {}
 
+export interface IColumnEditorOptions {
```

Comment:
Assuming this is supposed to be used in the Query builder as well, there seems to be a considerable effort to not use column wording there, so we should consider having FieldEditorOptions and if needed alias those for the column like the pipe args